packages/evm-adapter/test/Helio.ts

```diff
@@ -209,12 +215,19 @@ describe('Helio protocol', function () {
 
       sleep(15 * 1000);
       const recipientBalance = await provider.getBalance(recipient.address);
+      console.log(
```

Comment:
did you forget this ?

Reply:
This is test, I'm console logging so I get info how the balances are changing